test/e2e/util/fixture.go

```diff
@@ -229,6 +236,7 @@ func teardown(ctx context.Context, t *testing.T, client crclient.Client, hc *hyp
 		for _, metricName := range []string{
 			hostedcluster.HostedClusterDeletionDurationMetricName,
 			hostedcluster.HostedClusterGuestCloudResourcesDeletionDurationMetricName,
+			HypershiftOperatorInfo,
```

Comment:
can we be consistent with the other naming i.e HypershiftOperatorInfo -> HypershiftOperatorInfoName?